### PR TITLE
chore(phrases): remove unnecessary exclamation points from phrases

### PR DESCRIPTION
**Pull request by charIeszhao**
## Summary

Remove unnecessary exclamation points from all translation scripts.

## Testing

N/A

## Checklist

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/phrases/src/locales/fr/translation/admin-console/profile.ts [hunk 1]
```diff
@@ -69,11 +69,11 @@ const profile = {
   change_name: 'Modifier le nom',
   change_username: "Modifier le nom d'utilisateur",
   set_name: 'Définir le nom',
-  email_changed: 'Email modifié !',
-  password_changed: 'Mot de passe modifié !',
-  updated: '{{target}} mis à jour !',
-  linked: '{{target}} lié !',
-  unlinked: '{{target}} détaché !',
+  email_changed: 'Email modifié .',
```
Comment (gao-sun) [inline packages/phrases/src/locales/fr/translation/admin-console/profile.ts:72]: extra space before the dot?

File: packages/phrases/src/locales/pl-pl/translation/admin-console/applications.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ const applications = {
   select_application_type: 'Select an application type',
   no_application_type_selected: 'You haven’t selected any application type yet',
   application_created:
-    'The application {{name}} has been successfully created! \nNow finish your application settings.',
+    'The application {{name}} has been successfully created. \nNow finish your application settings.',
```
Comment (gao-sun) [inline packages/phrases/src/locales/pl-pl/translation/admin-console/applications.ts:14]: Extra space?

File: packages/phrases/src/locales/pt-pt/translation/admin-console/applications.ts [hunk 3]
```diff
@@ -11,7 +11,7 @@ const applications = {
   select_application_type: 'Selecione o tipo de aplicação',
   no_application_type_selected: 'Ainda não selecionou nenhum tipo de aplicação',
   application_created:
-    'A aplicação {{name}} foi criada com sucesso! \nAgora termine as configurações do seu aplicativo.',
+    'A aplicação {{name}} foi criada com sucesso. \nAgora termine as configurações do seu aplicativo.',
```
Comment (gao-sun) [inline packages/phrases/src/locales/pt-pt/translation/admin-console/applications.ts:14]: Extra space?

File: packages/phrases/src/locales/tr-tr/translation/admin-console/applications.ts [hunk 4]
```diff
@@ -11,7 +11,7 @@ const applications = {
   select_application_type: 'Uygulama tipi seçiniz',
   no_application_type_selected: 'Henüz bir uygulama tipi seçmediniz',
   application_created:
-    '{{name}} Uygulaması başarıyla oluşturuldu! \nŞimdi uygulama ayarlarını tamamlayın.',
+    '{{name}} Uygulaması başarıyla oluşturuldu. \nŞimdi uygulama ayarlarını tamamlayın.',
```
Comment (gao-sun) [inline packages/phrases/src/locales/tr-tr/translation/admin-console/applications.ts:14]: Ditto for this phrase, would you like to also check the remaining languages? Thanks

**Review by gao-sun — APPROVED** [review]

File: packages/phrases/src/locales/fr/translation/admin-console/profile.ts (hunk 1 above)
Comment (charIeszhao) [reply packages/phrases/src/locales/fr/translation/admin-console/profile.ts:72]: hmmm, will check it later

**Pull request merged**